modules/services/fnott.nix

```
@@ -0,0 +1,180 @@
+    anchor = [ "main" "anchor" ];
+    playSound = [ "main" "play-sound" ];
+    maxIconSize = [ "main" "max-icon-size" ];
+    iconTheme = [ "main" "icon-theme" ];
+    maxHeight = [ "main" "max-height" ];
+    minWidth = [ "main" "min-width" ];
+    maxWidth = [ "main" "max-width" ];
+  };
+
+in {
+  options = {
+    services.fnott = {
+      enable = mkEnableOption ''
+        fnott, a lightweight Wayland notification daemon for wlroots-based compositors.
+      '';
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.fnott;
+        defaultText = literalExample "pkgs.fnott";
+        description = "Package providing <command>fnott</command>.";
+      };
+
+      output = mkOption {
+        type = types.str;
+        description = ''
+          Show notifications on the specified output. If empty, notifications will
+          appear on the focused output. Requires the compositor to support the Wayland
+          protocol xdg-output-unstable-v1 version 2.
+
+          On
+          <citerefentry>
+            <refentrytitle>sway</refentrytitle>
+            <manvolnum>1</manvolnum>
+          </citerefentry>,
+          you can use <literal>swaymsg -t get_outputs</literal> to get a list of
+          avalaible outputs.
+        '';
+      };
+
+      anchor = mkOption {
+        type =
+          types.enum [ "top-right" "top-left" "bottom-right" "bottom-left" ];
+        description = ''
+          Show notifications at the specified position on the output.
+          Supported values are top-right, top-left, bottom-right,
+          and bottom-left.
+        '';
+      };
+
+      playSound = mkOption {
+        type = types.str;
+        defaultText =
+          literalExample "\${pkgs.alsa-utils}/bin/aplay \${filename}";
+        description = ''
+          Command to execute to play notification sounds.
+          ''${filename} will be expanded to the path
+          to the audio file to play.
+        '';
+      };
+
+      maxIconSize = mkOption {
+        type = types.int;
+        description = ''
+          Maximum icon size, in pixels. Icons larger than this will be scaled down.
+        '';
+      };
+
+      iconTheme = mkOption {
+        type = types.str;
+        description = ''
+          Icon theme to use when a notification has requested an non-embedded icon.
+        '';
+      };
+
+      maxHeight = mkOption {
+        type = types.int;
+        description = "Maximum notification height, in pixels.";
+      };
+
+      minWidth = mkOption {
+        type = types.int;
+        description = "Minimum notification width, in pixels.";
+      };
+
+      maxWidth = mkOption {
+        type = types.int;
+        description = "Maximum notification width, in pixels.";
+      };
+
+      settings = mkOption {
+        type = iniFormat.type;
+        default = { };
+        description = ''
+          Configuration written to
+          <filename>$XDG_CONFIG_HOME/fnott/fnott.ini</filename>.
+
```

To insert a paragraph break you need to add
```suggestion
          </para><para>
```

---

oh I see